src/components/Statistics/Statistics.tsx

```diff
@@ -1,12 +1,29 @@
-import Graph from '@components/Graph/Graph';
-import { Graphs } from '@components/Graph/Graph.style';
+import Graph from '@components/Graph';
 import Slice from '@components/Slice';
 import TextList from '@components/TextList';
 import { dataNumberFcu } from '@data';
 import { useMemo } from 'react';
 import useSWR from 'swr';
 import Band from './Band';
-import { Container } from './Statistics.style';
+import { Container, GraphsWrapper } from './Statistics.style';
+
+type ReturnApiMatomo = {
+  filters?: { date?: string };
+  nb_uniq_visitors?: string;
+  'Formulaire de test - Envoi'?: { nb_visits?: number | string };
```

Comment:
Would have been nice to have a type for nb_visits

Reply:
Sure 👍